<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1410</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textedit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>501</width>
     <height>671</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>3</width>
     <height>4</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>  Pattern Viewer Program</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>770</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>3</width>
     <height>4</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Keyword:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyword1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>770</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>770</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextbutton1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>810</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textedit2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>501</width>
     <height>671</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>770</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>3</width>
     <height>4</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Keyword:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyword2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>770</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>770</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextbutton2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>810</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>840</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Copyright Â©UNITEST Corporation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileSelect">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>760</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>File Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="filename">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>840</y>
     <width>1031</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>dfdsadasdasdasdasd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>800</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
